src/dms-preview/azext_dms/_help.py

```diff
@@ -39,6 +40,7 @@
             1) SQL -> SQLDB :: OfflineMigration
             2) MySQL -> AzureDbForMySql :: OnlineMigration
             3) PostgreSQL -> AzureDbForPostgreSQL :: OnlineMigration
+            4) MongoDB -> Cosmos DB (via MongoDB API) :: OfflineMigration
```

Comment:
Can you replace "Cosmos DB" with the target type? Eg, PostgreSQL -> AzureDbForPostgreSQL means that the source type is PostgreSQL and target type is AzureDbForPostgreSQL.

Reply:
Ok. But some extra text should be added somehow. The scenario is to enable migrating from MongoDB to Cosmos DB. But the actual type of both the source and the target is MongoDB, since we connect to a Comsos DB instance via their MongoDB API. I was trying to make that clear with this. If a user just saw "MongoDB -> MongoDB" they might not understand what's going on and what the intention of the scenario is. Your point is valid, I'll fiddle with the wording.